packages/client/src/dataunion/Contracts.ts

```diff
@@ -21,6 +22,7 @@ function validateAddress(name: string, address: EthereumAddress) {
     }
 }
 
+/* TODO: remove this class and inline its functions into DataUnion.ts */
```

Comment:
If you decide to make tickets about these issues (as discussed in the PR) it would be useful to refer to them in the comments, e.g.

```ts
/* TODO: ETH-1234, remove this class and inline its functions into DataUnion.ts */
```